core/utils/colour.js

```diff
@@ -211,3 +211,39 @@ Blockly.utils.colour.names = {
   'white': '#ffffff',
   'yellow': '#ffff00'
 };
+
+/**
+ * Parse a block colour from a number or string, as provided in a block
+ * definition.
+ * @param {number|string} colour HSV hue value (0 to 360), #RRGGBB string,
+ *     or a message reference string pointing to one of those two values.
+ * @return {{hue: ?number, colour: string}} An object containing the colour as
```

Comment:
update @return type to ``{{hue: ?number, hex: string}}`` to reflect update below.

Reply:
Done.